### DISCUSSION TITLE
React-toastify toasts cannot be clicked inside a Dialog component

### DISCUSSION THREAD

**Discussion by Supcar27** [tailwindlabs/headlessui#3224] [Help] [answered]
headlessui React v1.7

I used the Dialog component to make a drawer component and i noticed when the drawer is open i cannot click on toast messages coming from react-toastify, and if there is a button on the drawer in the spot where the toast message appears when clicked it will actually click the drawer button like the toast is not there

Any ideas what can cause this effect and how to fix this?

Thanks in advance!

**Comment by Supcar27** [comment, marked as answer]
for anyone having this issue i solved this by having more then 1 toast container. What i found is as long as the container is within the Dialog (Drawer) the toasts can be interacted with but when the toast container is just in the App.tsx then when drawer is open toasts cannot be interacted with.

Hope this helps!

**Reply by tgwasira** [reply]
Very helpful. Thanks.

**Reply by nicholasmanske** [reply]
Thank you! This was very painful until I tried your solution
